### ISSUE TITLE
preparation for a production deployment

### ISSUE THREAD

**Issue by eauge** [oasislabs/developer-gateway#51]
This is a list of tasks that need to be done before we can release the developer-gateway and deploy it to production

Needed:
 - [x] Redis implementation production ready (https://github.com/oasislabs/developer-gateway/issues/106)
 - [missed ] Increase unit test coverage to 80%
 - [missed ] Have e2e tests with decent coverage (https://github.com/oasislabs/developer-gateway/issues/50)
 - [x] Report usage and health metrics (https://github.com/oasislabs/developer-gateway/issues/97)
 - [x] Deployment charts and scripts
 - [x] Policy definition (https://github.com/oasislabs/developer-gateway/issues/4)
 - [x] Wallet accounting (https://github.com/oasislabs/developer-gateway/issues/95)
 - [x] Dashboard integration (https://github.com/oasislabs/developer-gateway/issues/96)
 - [x] Documentation (https://github.com/oasislabs/developer-gateway/issues/32, https://github.com/oasislabs/developer-gateway/issues/102, https://github.com/oasislabs/developer-gateway/issues/101)
 - [x] Security review (https://github.com/oasislabs/developer-gateway/issues/32)

Nice to have:
 - [missed ] Implementation of ekiden backend (https://github.com/oasislabs/developer-gateway/issues/98)
